File: Problema01/Java/Main.java
import java.io.BufferedReader;
import java.io.InputStreamReader;

class Main {
  public static void main(String[] args) {
	// Objects & Vars/Cons
	BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
	float a = 0, b = 0, c = 0;
	float result;

	// Inputs
    try{
		print("Ingrese tres números enteros: \n");
		a = Float.parseFloat(br.readLine());
		b = Float.parseFloat(br.readLine());
		c = Float.parseFloat(br.readLine());
    }catch(Exception e){
      	print("Se generó la siguiente excepción: " + e.getMessage());
    }
	
	// Process & Outputs
	print("\n\n--- RESULTADOS ---");

	result = (a + b) * c;
	print("\na + b * c = " + result);

	result = (b + a) % c;
	print("\nb + a mod c = " + result);

	result = b / (a % c);
	print("\nb / a mod c = " + result);
  }

	public static void print(String s){
    	System.out.print(s);
  	}
}

File: Problema01/Python/main.py
# Variables
a = b = c = result = 0

# Inputs
try:
  a = int(input("Ingresa el valor de a: "))
  b = int(input("Ingresa el valor de b: "))
  c = int(input("Ingresa el valor de c: "))
except Exception as e:
  print("\nSe produjo la siguiente excepción: " + str(e))

# Process & Outputs
print("\n--- RESULTADOS ---")

result = (a + b) * c
print("a + b * c = " + str(result))

result = (b + a) % c
print("b + a mod c = " + str(result))

result = b / (a % c)
print("b / a mod c = " + str(result))
